### PR TITLE
add Indigitization link

### PR DESCRIPTION
**Pull request by privatezero**
good info about tapes!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ If you have never contributed to GitHub before, [here is a guide](http://ablwr.g
 - [Collective Access Community: Guides and Workflows](http://collectiveaccesscommunity.org/category/guides/)
 - [Digital Processing, Bentley Historical Library](https://sites.google.com/a/umich.edu/bhl-archival-curation/processing-archival-collections/08-digital-processing): Workflow for ingesting digital objects at the Bentley Historial Library.
 - [Format Migrations at Harvard](http://blogs.loc.gov/thesignal/2015/04/format-migrations-at-harvard-library-an-ndsr-project-update/)
+- [Indigitization](http://www.indigitization.ca/indigitization-toolkit/introduction/) A 'toolkit' of workflows focussed on supporting First Nations communities' digitization of cultural resources.  Contains detailed walkthroughs for Cassette digitization as well as metadata best practices.
```
Comment (ablwr) [inline README.md:21]: "focussed"!

**Review by ablwr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (privatezero) [reply README.md:21]: Focussed is a perfectly legitimate spelling for that word! Not a typo!!

Comment (ablwr) [reply README.md:21]: Whoops!

Comment (privatezero) [reply README.md:21]: next you will want me to add in some unnecessary oxford commas won't you ;-P

**Review by ablwr — APPROVED** [review]

**Comment by ablwr** [comment]
Thank youuuu

File: README.md (hunk 1 above)
Comment (ablwr) [reply README.md:21]: Mr Weaver, you are the one adding an extra unnecessary letter!

Comment (privatezero) [reply README.md:21]: Touche!

Comment (retokromer) [inline README.md:21]: Don't use double spaces. Single are sufficient.

Comment (privatezero) [reply README.md:21]: I suppose my personal taste in typographic style must bow to the consistency of the document!  I'll edit that out!

Comment (retokromer) [reply README.md:21]: It was used in the US for typewriting.

Comment (privatezero) [reply README.md:21]: The legacies of early 90s public schooling are hard to overcome! :-P

**Pull request merged**